tests/keras/engine/test_topology.py

```diff
@@ -13,7 +13,8 @@
 
 
 skipif_no_tf_gpu = pytest.mark.skipif(
-    (K.backend() != 'tensorflow') or (not K.tensorflow_backend._get_available_gpus()),
+    ((K.backend() != 'tensorflow') or
+     (not K.tensorflow_backend._get_available_gpus())),
```

Comment:
Suggest to put operator in next line and remove redundant parentheses.
```python
    (K.backend() != 'tensorflow'
        or not K.tensorflow_backend._get_available_gpus()),
```

Reply:
Thank you for the comment, @joelthchao. That's good.